docs/docker_linux.rst

```diff
@@ -15,7 +15,7 @@ On Ubuntu, you can install Docker with the following commands:
    sudo apt update
    sudo apt install docker.io -y
 
-On Azure Linux, you can install Docker with the following commands:
+On Azure Linux (Mariner) 2.0 or 3.0, you can install Docker with the following commands:
```

Comment:
It's not here. This document is for using docker on Azure Linux. But there is another document is to install into Linux. Please update that document. This document doesn't need to be changed.